src/Java.Interop.Tools.TypeNameMappings/Java.Interop.Tools.TypeNameMappings/JavaNativeTypeManager.cs

```diff
@@ -58,8 +58,8 @@ public static JniTypeName Parse (string jniType)
 
 		public static IEnumerable<JniTypeName> FromSignature (string signature)
 		{
-			if (signature.StartsWith ("(")) {
```

Comment:
Should this instead use [`signature.StartsWith('(')`](https://docs.microsoft.com/en-us/dotnet/api/system.string.startswith?view=net-5.0#System_String_StartsWith_System_Char_), which "mirrors" changes elsewhere which use `char` instead of `string, StringComparison` overloads? For example the `signature.IndexOf(')')` change on the following line?

Likewise many of the other changes that only deal with a single character…

Reply:
Ideally yes, but those overloads are introduced in NET Core 2.0/NET Standard 2.1 so they are not available to us.